### PR TITLE
remove wikidevelopermozillaorg links (pt-br)

### PR DESCRIPTION
**Pull request by jovinoguerrero**
Part of issue #1440
@peterbe

### CODE REVIEW

File: files/pt-br/learn/html/multimedia_and_embedding/video_and_audio_content/index.html [hunk 1]
```diff
@@ -220,7 +220,7 @@ <h2 id="Exibindo_trilhas_de_texto_em_vídeo">Exibindo trilhas de texto em vídeo
  <li>Da mesma forma, em ambientes em que a reprodução do áudio seria uma distração ou perturbação (como em uma biblioteca ou quando um parceiro está tentando dormir), ter legendas pode ser muito útil.</li>
 </ul>
 
-<p>Não seria bom poder fornecer a essas pessoas uma transcrição das palavras que estão sendo ditas no áudio / vídeo? Bem, graças ao vídeo HTML5, você pode, com o formato <a href="https://wiki.developer.mozilla.org/en-US/docs/Web/API/WebVTT_API">WebVTT </a>e o elemento {{htmlelement ("track")}}.</p>
+<p>Não seria bom poder fornecer a essas pessoas uma transcrição das palavras que estão sendo ditas no áudio / vídeo? Bem, graças ao vídeo HTML5, você pode, com o formato <a href="/en-US/docs/Web/API/WebVTT_API">WebVTT </a>e o elemento {{htmlelement ("track")}}.</p>
```
Comment (peterbe) [inline files/pt-br/learn/html/multimedia_and_embedding/video_and_audio_content/index.html:223]: Ideally, all links should be `pt-BR` actually. 
```suggestion
<p>Não seria bom poder fornecer a essas pessoas uma transcrição das palavras que estão sendo ditas no áudio / vídeo? Bem, graças ao vídeo HTML5, você pode, com o formato <a href="/pt-BR/docs/Web/API/WebVTT_API">WebVTT </a>e o elemento {{htmlelement ("track")}}.</p>
```

When Yari builds, it'll check that and if the slug `Web/API/WebVTT_API` hasn't been translated yet, it automatically turns the link into `<a href="/en-US/docs/Web/API/WebVTT_API">`

But it's more important to get rid of these old absolute URLs for now.

Comment (jovinoguerrero) [reply files/pt-br/learn/html/multimedia_and_embedding/video_and_audio_content/index.html:223]: @peterbe All right! I think I can help with that too 😉

**Comment by peterbe** [comment]
@jovinoguerrero When you merge in the latest `main` you're going to get conflicts. What I propose you do is just run:
```
git rm files/pt-br/orphaned/mdn/editor/basics/index.html files/pt-br/orphaned/web/api/document_object_model/events/index.html
git commit -a -m "deleted conflicting files"
```
does that make sense?

**Comment by josielrocha** [comment]
Esse PR está há bastante tempo sem interação e, dadas todas as mudanças que já tivemos no código não faz mais sentido.
Fechando para evitar de ficarmos verificando se houve alguma interação todas as vezes.

**Pull request closed without merging**